MCprep_addon/materials/prep.py

```diff
@@ -119,6 +124,12 @@ def pack_formats(self, context):
 		description="Change the pack format when using a PBR resource pack.",
 		items=pack_formats
 	)
+	blendMode = bpy.props.EnumProperty(
```

Comment:
Per further above, does this completely replace the need for having the config file for your optimizer settings? (or are you planning to do more with that? either way is fine, just would rather not merge in code that "might get used" as it's hard to tell what's safe to delete at any given moment in time)

Reply:
JSON config was replaced with an advanced panel that only shows up if the user enables it in user preferences. Optimizer configuration allows the user to find tune and control the internal parts of the optimizer (such as max and minimum bounces, volumetric stepping rate, etc.)

Reply:
Ok I think that's ok then, I was sort of thinking having a config file for a specific feature felt a little excessive and not that user friendly anyways.